### ISSUE TITLE
-Clean Parameter does not work

### ISSUE THREAD

**Issue by ah-uk** [aaronparker/LatestUpdate#29]
# Expected Behavior

All packages are removed from packagepath when the -clean parameter is used.

# Current Behavior

No packages are removed.

# Failure Information (for bugs)

When an older update is present in the packagepath e.g an earlier cumulative update in the month (due to there being twice a month sometimes) it will error with the following 

```
VERBOSE: Destination is DS004:\Packages\Windows_10_1803_x64
VERBOSE: Performing the operation "Remove package" on target "Package_for_KB4485449 neutral amd64 17134.580.1.0".
Get-ChildItem : This row has been removed from a table and does not have any data.  BeginEdit() will allow creation of new data in this row.
At C:\Program Files\WindowsPowerShell\Modules\LatestUpdate\2.3.1.81\Public\Import-LatestUpdate.ps1:100 char:13
+             Get-ChildItem | Where-Object { $_.Name -like "Package*" } ...
+             ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Get-ChildItem], RowNotInTableException
    + FullyQualifiedErrorId : ExceptionCaught,Microsoft.PowerShell.Commands.GetChildItemCommand
```

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Ensure that an older update is present (Imported) in the package path. This must be one that is not currently offered by Get-LatestUpdate.
2. Run 'Import-LatestUpdate -UpdatePath "C:\Temp\Updates" -DeployRoot "\\SERVER\DeploymentShare$" -PackagePath "Windows_10_1803_x64" -Clean -Verbose'
3. It will then error upon trying to clean up the first package found in the package path. No further cleaning is performed. Meaning over time packages will build up in MDT.

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* **LatestUpdate Version**: 2.3.1.81
* **PowerShell Version**: 5.1 Build 14409 Revision 1018
* **Operating System**: Windows Server 2012 R2

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by aaronparker** [comment]
Fixed in commit 3ae65830d55273082484328592affc4860a126e7

**Comment by aaronparker** [comment]
Fixed in release v2.4.0.89.
